.github/renovate.json

```diff
@@ -0,0 +1,16 @@
+{
+  "extends": [
+    "config:base",
+    ":enablePreCommit",
+    ":preserveSemverRanges",
+    ":disableDependencyDashboard"
+  ],
+  "labels": ["dependencies"],
+  "packageRules": [
+    {
+      "matchManagers": ["github-actions"],
+      "addLabels": ["ci"]
+    }
+  ],
+  "ignorePaths": ["*-client/**"]
```

Comment:
Which #15, we might want to track the javascript client to bump some dependencies, same goes for the python client.

Should we handle this in a futur PR ?

Reply:
Yup, we will handle this in a future PR.